modules/tinymce/src/plugins/link/test/ts/browser/SelectedTextLinkTest.ts

```diff
@@ -40,6 +40,16 @@ describe('browser.tinymce.plugins.link.SelectedTextLinkTest', () => {
     existence(SugarBody.body(), '.tox-label:contains("Text to display")');
   };
 
+  it('TBA: inserting new link should move cursor behind the new link', async () => {
+    const editor = hook.editor();
+    editor.setContent('<p>word</p>');
+    TinySelections.setSelection(editor, [ 0 ], 0, [ 0 ], 1); // <p>[word]</p>
```

Comment:
Actually this selects `[<p>word</p>]` - a more realistic user selection would be
```suggestion
    TinySelections.setSelection(editor, [ 0, 0 ], 0, [ 0, 0 ], 4); // <p>[word]</p>
```

Reply:
Ah yes, I was wondering about that. Both whether it's correct (and it wasn't) and which way round would be the "realistic" way, i.e. what happens if someone manually selects the word. Thanks for the suggestion, I've updated the PR.